### DISCUSSION TITLE
Unexpected token error (JSX node) in with-typescript example when using symlinked project directory

### DISCUSSION THREAD

**Discussion by transiient** [vercel/next.js#14185] [Help]
# Bug report

## Describe the bug

I'm using Powershell inside Windows Terminal. I've got a Documents folder on another hard drive which is symlinked via a `Documents-SL` link.

I navigated through to my dev projects directory via this symlink and got this error when launching my Next.JS project:

```
error - E:/Users/offic/Documents/_Dev/_Web/practice-strapi-website/frontend/pages/index.tsx 5:2
Module parse failed: Unexpected token (5:2)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|
| const IndexPage = () => (
>   <Layout title="Home | Next.js + TypeScript Example">
|     <h1>Hello Next.js �</h1>
|     <p>
```

## To Reproduce

**You must have a directory symlink to reproduce this bug.** I am using a `Documents` folder on another hard drive.

1. `cd` into a symlink directory - (eg: `cd C:\Users\abcde\Documents-Symlink\Projects`)
2. Run `yarn create next-app frontend`
3. Select the `with-typescript` example project as a template
4. `cd` into the `./frontend` directory
5. Run `yarn dev`
6. Navigate to `http://localhost:3000/`

## Expected behavior

Expected the page to load. Instead got an error page.

## System information

- OS: Windows 10 x64 [Build 18363.900]
- Browser: Mozilla Firefox 77.0.1
- Version of Next.js: 9.4.4
- Version of Node.js: 14.3.0

## Additional Context

I don't think this is necessary a template issue, or even a Next JS issue necessarily, but I'm not sure where the issue stems and someone else may have more luck with debugging/tracing.

**Comment by chrisabrams** [comment]
Facing the same issue when upgrading to any minor version after 9.3.
